client/verta/verta/_registry/modelversion.py

```diff
@@ -122,9 +127,12 @@ def log_model(self, model, overwrite=False):
             _CommonCommonService.ArtifactTypeEnum.MODEL,
         )
 
+    def get_model(self):
+        return _artifact_utils.deserialize_model(self._get_artifact("model", _CommonCommonService.ArtifactTypeEnum.MODEL))
```

Comment:
Since I'm requesting another minor change anyway...
Could we divide this into two lines? It'd help with readability.

Reply:
Updated.